### PR TITLE
#103 css linter

### PR DESCRIPTION
**Pull request by Draeggon**

### CODE REVIEW

**Review by landerneRPi — APPROVED** [review]

File: doc/tools/stylelint.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# stylelint
+
+## install stylelint
+```sh
+  npm install --save-dev stylelint stylelint-config-standard
```
Comment (Benvii) [inline doc/tools/stylelint.md:5]: Maybe tell the user to go to /front directory, and what about moving this documentation to the front directory also ?

File: doc/tools/stylelint.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+```
+
+## how to play
+
+ Run stylelint and autofix:
+```sh
+  npx stylelint --fix "src/css/*.css"
```
Comment (Benvii) [inline doc/tools/stylelint.md:12]: Tell the user he as to be in the same folder (where the npm install command was run)

**Review by Benvii — APPROVED** [review]
Maybe move this documentation ? Or link it from the front README

**Comment by js-all** [comment]
You do realize that piling up linters to the moon won't build an application ?

**Comment by Draeggon** [comment]
@js-all  yeahhh ... i know but the issue is here for a long time and is in V1 project so ... the next will be vue linter :joy:

**Review by Benvii — APPROVED** [review]
Good

**Pull request merged**